README.rst

```diff
@@ -1,11 +1,39 @@
 What is borg-import?
 ====================
 
-borg-import can import backups from some other backup software, currently supported are:
+A set of scripts to import from other backup software into the archive format used by `BorgBackup <https://github.com/borgbackup/borg>`_.
```

Comment:
an `archive` (in borg terms) is the result of a `borg create` operation.

but there is also a special `repository` format (and only that is seen by the user when looking at a borg repo directly).

I don't think it is useful to talk about archives or repos here, maybe rather `storage format` or `backup format`. or just keep the previous phrasing.

`a set of scripts` also sounds worse than it is, maybe rather `a tool`?